tests/tier2/tc_2066_validate_cluster_name_with_special_char.py

```diff
@@ -0,0 +1,63 @@
+# coding:utf-8
+from virt_who import *
+from virt_who.base import Base
+from virt_who.register import Register
+from virt_who.testing import Testing
+
+
+class Testcase(Testing):
+    def test_run(self):
+        self.vw_case_info(os.path.basename(__file__), case_id='RHEL-196078')
+        hypervisor_type = self.get_config('hypervisor_type')
+        if hypervisor_type != 'esx':
+            self.vw_case_skip(hypervisor_type)
+        self.vw_case_init()
+
+        # Case Config
+        results = dict()
+        config = self.get_hypervisor_config()
+        ssh_hypervisor = config['ssh_hypervisor']
+        register_config = self.get_register_config()
+        server = register_config['server']
+        ssh_user = register_config['ssh_user']
+        ssh_passwd = register_config['ssh_passwd']
+        ssh_register = {"host": server, "username": ssh_user, "password": ssh_passwd}
+        admin_user = register_config['username']
+        admin_passwd = register_config['password']
+        config_name = "virtwho-config"
+        config_file = "/etc/virt-who.d/{0}.conf".format(config_name)
```

Comment:
should we enable the debug mode for console output?

Reply:
Yes, we should. Thank you.